.editorconfig

```diff
@@ -111,6 +111,9 @@ dotnet_style_require_accessibility_modifiers = omit_if_default:warning
 #raise a warning on build when default access modifiers are explicitly specified
 dotnet_diagnostic.IDE0040.severity = warning
 
+dotnet_diagnostic.CA1825.severity = warning    # Avoid unnecessary zero-length array allocations.
```

Comment:
The rest of the file places comments on the line before the rule. Can we please keep a consistent style?

Reply:
To be honest I would really like to move away from the current style in this file and towards something more like the style in `OpenRA.ruleset`, where everything is neat and tidy, while here we have several comment rule violations - no empty line above, no leading space and no leading capital letter 😐 
But we should probably discuss the still-existing separation between style rules in `.editorconfig` and in `OpenRA.ruleset`.

Reply:
Sure, I have no problem with changing all the other comments to match this - but lets keep it consistent.